index.html

```diff
@@ -76,17 +76,17 @@ <h5>LET'S CONNECT</h5>
       </section>  
    </section>
   </main>
-  
-      <!--Work Section -1  -->
-   
-  <section class="work-section"  id="home">
-        <div class="work-container">
-            <div class="img img-1">
-              <img src="./images/Desktop/Grid-1/Snapshoot_Portfolio_1.svg" alt="first-work">
-              <div class="content">
-                <h2>Tonic</h2>
-              <div class="canopy">
-                  <h4>Canopy</h4>
+
+
+
+ <section class="work-modal">
+      <div class="modal" id="modal">
+                 <div class="modal-header">
+                    <h3 class="modal-title">title</h3>
+                    <button class="close-button">&times;</button>
+                 </div>
+                <div class="canopy modal-subTitle">
+                  <h4 id="sub-canopy">Canopy</h4>
```

Comment:
- [ ] I think it would be nice to dynamically generate and populate the contents of the modal in javascript based on which project is selected. Currently, even though the display is set to none the modal is statically rendered when the page first loads, and content is changed every time project is selected.

Reply:
Thank you I have made the required changes.